.gitattributes

```diff
@@ -1,7 +1,6 @@
 *.jpg filter=lfs diff=lfs merge=lfs -text
 *.png filter=lfs diff=lfs merge=lfs -text
 *.pdf filter=lfs diff=lfs merge=lfs -text
-*.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
Why would you this? This line guarantees that all PNG files are added to LFS and not directly to git.